Project_source/app/src/main/java/android/clicker/school_live_simulator/PlayerState.kt

```diff
@@ -0,0 +1,11 @@
+package android.clicker.school_live_simulator
+/**
+ * @author      Andrey Novikov andrewnovicov@gmail.com
+ * @version     1.0                 (current version number of program)
+ * @since       1.0          (the version of the package this class was first added to)
+ */
+abstract class PlayerState {
+    protected abstract val reduce_school_performance_percent: Int
+    protected abstract val reduce_happines_percent: Int
+    protected abstract val reduce_satiety_percent: Int
```

Comment:
Please, add some note commentaries for what this class and when these values should be used.
Set 0 for all fields in abstract class.

Reply:
Added commentaries.
I have a question about setting 0 for all fields in abstract class. Abstract values cannot be initialized.

Reply:
@Ferrociraptus check Andrey's commentary please..

Reply:
@Endray952, firstly, these variables should be var, not val. Secondly, you should remove abstract modificator before variables and set initializing values.

Reply:
Fixed the way I see it. Waiting for response.